mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -1199,6 +1223,40 @@ resource "google_sql_database_instance" "instance-with-ad" {
 }`, networkName, addressRangeName, databaseName, rootPassword, adDomainName)
 }
 
+func testGoogleSqlDatabaseInstance_SqlServerAuditConfig(databaseName, rootPassword, bucketName string) string {
```

Comment:
The test failed with the following error: 
```
        Error: Error, failed to create instance because the network doesn't have at least 1 private services connection. Please see https://cloud.google.com/sql/docs/mysql/private-ip#network_requirements for how to create this connection.
```